AISKU/src/Initialization.ts

```diff
@@ -57,7 +54,7 @@ export interface Snippet {
 }
 
 export interface IApplicationInsights extends IAppInsights, IDependenciesPlugin, IPropertiesPlugin {
-    appInsights: ApplicationInsights;
+    appInsights: AnalyticsPlugin;
```

Comment:
I am just curious, `ApplicationInsights` includes deprecated instance, so we switch to `analyticsplugin` directly here?

Reply:
For this one I renamed the exported object from "ApplicationInsights" to "AnalyticsPlugin" a few releases ago and as part of this (for backward compatibility) it also exported the "AnalyticsPlugin" as "ApplicationInsights" (so 2 names referencing the same class).